jekyll/_cci2/workflows.md

```diff
@@ -172,21 +172,24 @@ Following is a screenshot of the Approval dialog box that appears when you click
 
 ## Scheduling a Workflow
 
-Workflows that are resource-intensive or that generate reports may be run on a schedule rather than on every commit. This feature is configured by adding a scheduled trigger to the configuration of the workflow. 
+It can be inefficient and expensive
+to run a workflow for every commit for every branch.
+Instead,
+you can schedule a workflow
+to run at a certain time for specific branches.
 
-Configure a workflow to run on a set schedule by using the `triggers:` key with the `schedule` option. The `triggers` key is **only** added under your `workflow` key. This feature enables you to schedule a workflow run by using `cron` syntax to represent Coordinated Universal Time (UTC) for specified branches. 
```

Comment:
no need to remove, add back

Reply:
Much of this information has only been moved, not removed:

"Configure a workflow to run on a set schedule by using the `triggers` key with the `schedule` option." --> Moved to lines 188-190.
"The `triggers` key is **only** added under your `workflow` key. --> Removed. Better explained in the linked Configuration Reference. Also not technically correct because the "workflow" key will be a named key.
"This feature enables you to schedule a workflow run by using `cron` syntax to represent Coordinated Universal Time (UTC) for specified branches." --> Moved to lines 220-226. Not entirely correct to call it `cron` syntax. Instead, the value of the `cron` key should be a valid crontab entry.

Reply:
Sorry again, but please add this content and link back, I'm not going to revisit all of these decisions again. Your additions are great and not in conflict with what was already there! Thanks!